### PR TITLE
Update Clickhouse to 19.7.3.9

### PR DESCRIPTION
**Pull request by gongled**
Hi @andyone,

I ask you to build new package for Clickhouse with the following changes:
- [x] Update Clickhouse to v19.7.3.9-stable.
- [x] Logrotate configuration (rotate once a day using SIGHUP signal)
- [x] Fix `No such file or directory` during on compilation (use `GLIBC_COMPATIBILITY=OFF` option) (https://github.com/yandex/ClickHouse/issues/5043).
- [x] Add dynamic linking with `librdkafka`.
- [x] Add dynamic linking with `liblz4`.

### CODE REVIEW

File: specs/clickhouse/clickhouse.spec [hunk 1]
```diff
@@ -319,7 +327,11 @@ fi
 ################################################################################
 
 %changelog
-* Tue Apr 09 2019 Anton Novojilov <andy@essentialkaos.com> - 19.4.3.11-0
+* Wed Jun 05 2019 Gleb Goncharov <g.goncharov@fun-box.ru> - 19.7.3.9-0
+- Updated to the latest release
+- Added logrotate configuration
+
+* Tue Apr 09 2019 Anton Novojilov <andy@essentialkaos.com> - 19.7.3.9-0
```
Comment (andyone) [inline specs/clickhouse/clickhouse.spec]: Not sure that this record is correct

Comment (gongled) [reply specs/clickhouse/clickhouse.spec]: Oops, my bad. Auto replacement.

**Comment by andyone** [comment]
Looks good, merged. Thanks!

**Comment by andyone** [comment]
The package has been built and added to `kaos-testing` repository.

**Pull request merged**